cypress/e2e/challenges.cy.js

```diff
@@ -0,0 +1,70 @@
+import ChallengesPage from '../pages/challengesPage'
+const challengesPage = new ChallengesPage()
+
+describe('example to-do app', () => {
+  beforeEach(() => {
+    cy.visit('localhost:8080')
+  })
+
+  it('Check all challenges load correctly', () => {
+    cy.wrap([0, 1, 2, 3, 4, 8, 12, 13, 14, 15, 16, 17, 18, 19]).each((challengeNum) => {
+      cy.visit('localhost:8080')
```

Comment:
Can you make the host and port configurable? Fixed combination is repeated in the test and would be helpful if this can be configured in one place.

Reply:
This has been added now in the cypress config